src/index.html

```diff
@@ -1,17 +1,164 @@
+            Name:
+          <input
+            type="text"
+            name="name" 
+            autocomplete="off"
+            required>
+        </label>
+          
+        <label class="form-field">
+            How old are You?
+          <input 
+            type="number"
+            name="age" 
+            value="12" 
+            min="1"
+            max="100">
+        </label>
+  
+        <label class="form-field">
+            Full date of birth:
+          <input 
+            type="date"
+            name="birth">
+        </label>
+  
+        <label class="form-field">
+            I accept the term of the agreement:
+          <input 
+            type="checkbox"
+            name="term" 
+            required
+            checked>
+        </label>
+              
+      </fieldset>
+  
+      <fieldset>
+        <legend>Registration</legend>
+  
+        <label class="form-field">
+          E-mail:
+          <input 
+            type="email" 
+            placeholder="email@example.com" 
+            required>
+        </label>
+  
+        <label class="form-field">
+          Password:
+          <input 
+            type="password" 
+            required 
+            maxlength="30" 
+            minlength="8">
+        </label>
+      </fieldset>
+  
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+          
+        <div class="form-field">
+            Do you love cats?
+
+          <label >
+            <input 
+            type="radio" 
+            name="radio-button" 
+            value="yes">
+            - yes
+          </label>
+
+          <label>
+            <input 
+            type="radio" 
+            name="radio-button" 
+            value="no">
+            - no
+          </label>
+
+        </div>
+        <label class="form-field">
+           What is your favorite color?
+          <input 
+            type="color" 
+            name="color">
+        </label>
+
+        <label class="form-field">
+           What time do you go to bed?
+          <input 
+            type="time" 
+            name="time">
+        </label>
+
+        <label class="form-field">
+           What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="car 1">Audi</option>
+            <option value="car 2">Renault</option>
+            <option value="car 3">Jeep</option>
+            <option value="car 4">Peugeot</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+            How do you rate our work?
+          <input 
+            type="range"
+            name="range"
+            max="10"
+            min="0"
+            step="1">
+        </label>
+
+      </fieldset>
+  
+      <fieldset>
+        <legend>Additional info</legend>
+              
+        <label class="form-field">Comments:
+          <textarea 
+          name="comments" 
+          cols="30" 
+          rows="3"></textarea>
```

Comment:
```suggestion
          <textarea 
            name="comments" 
            cols="30" 
            rows="3"
          ></textarea>
```
Про правильні відступи і форматування можна почитати тут
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation